src/OpenTracing/Span.php

```diff
@@ -15,10 +15,13 @@ public function operationName();
     public function context();
 
     /**
+     * Finishes the span and locks it so it becomes immutable. As an implementor, make
```

Comment:
I don't think it has to "lock" it (as in all methods now throw exceptions), but I read that after `finish()` you should not use the Span anymore.